content/api/commands/mount.md

```diff
@@ -0,0 +1,226 @@
+
+If you attempt to use `cy.mount` before creating it, you will get a warning:
+
+<img src="/_nuxt/assets/img/guides/component-testing/cy-mount-must-be-implemented.png" alt="cy.mount must be implemented by the user." />
+
+Let's take a look at how to implement the command.
+
+## Creating a New `cy.mount` Command
+
+To use `cy.mount` you will need to add a
+[custom command](/api/cypress-api/custom-commands) to the commands file. Below
+are examples that you can start with for your commands:
+
+:::react-vue-example
+
+```js
+import { mount } from '@cypress/react'
+Cypress.Commands.overwrite('mount', (jsx, options) => {
+  // Wrap any parent components needed
+  // ie: return mount(<MyProvider>{jsx}</MyProvider>, options)
+  return mount(jsx, options)
+})
+```
+
+```js
+import { mount } from '@cypress/vue'
+Cypress.Commands.overwrite('mount', (comp, options = {}) => {
+  // Setup options object
+  options.global = options.global || {}
+  options.global.stubs = options.global.stubs || {}
+  options.global.stubs['transition'] = false
+  options.global.components = options.global.components || {}
+  options.global.plugins = options.global.plugins || []
+
+  /* Add any global plugins */
+  // options.global.plugins.push({
+  //   install(app) {
+  //     app.use(MyPlugin);
+  //   },
+  // });
+
+  /* Add any global components */
+  // options.global.components['Button'] = Button;
+
+  return mount(comp, options)
+})
+```
+
+:::
+
+See [examples](#Examples) below for practical use cases.
+
+## Adding TypeScript Typings for `cy.mount` Commands
+
+When working in
+[TypeScript](https://docs.cypress.io/guides/tooling/typescript-support), you
+will need to add some custom typings for your commands to get code completion
+and avoid any TypeScript errors. Below are the typings for the above examples
+for React and Vue:
+
+:::react-vue-example
+
+```ts
+// cypress.d.ts
+import { MountOptions, MountReturn } from '@cypress/react'
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /**
+       * Mounts a React node
+       * @param jsx React Node to mount
+       * @param options Additional options to pass into mount
+       */
+      mount(
+        jsx: React.ReactNode,
+        options?: MountOptions
+      ): Cypress.Chainable<MountReturn>
+    }
+  }
+}
+```
+
+```ts
+import { mount } from '@cypress/vue'
+type MountParams = Parameters<typeof mount>
+
+declare global {
+  namespace Cypress {
+    type ComponentParam = MountParams[0] | JSX.Element
+    type OptionsParam = MountParams[1]
+    interface Chainable {
+      /**
+       * Helper mount function for Vue Components
+       * @param component Vue Component or JSX Element to mount
+       * @param options Options passed to Vue Test Utils
+       */
+      mount(component: ComponentParam, options?: OptionsParam): Chainable<any>
+    }
+  }
+}
+```
+
+:::
+
+The typings will need to be in a location that any code can access in the
```

Comment:
I feel like the `cypress.d.ts` file should be mentioned before the code example. Also, if say do something like the following directly before the code example, you won't need the `// cypress.d.ts
` comment at the top of the code example:

> Add the following to your `cypress.d.ts` file:

The "updating any tsconfig.json files to reference the typings file in the include option:" bit with the code example could be added as an alert/note, but you could also just link over to https://docs.cypress.io/guides/tooling/typescript-support#Types-for-custom-commands and maybe add that note there, to maximize its visibility.